Assets/Scripts/PlayerController.cs

```diff
@@ -357,7 +357,13 @@ private void Meow()
     playerAudio.PlayMeow();
   }
 
-  private void UsePaw() { Debug.Log("UsePaw"); }
+  private void UsePaw()
+  {
+    Debug.Log("UsePaw");
+    gameObject.transform.Translate(Vector3.forward * 2, Space.Self);
```

Comment:
The movement on this feels a little bit abrupt for the distance covered. I changed the `2` to `1.2f` locally and it felt somewhat more natural.

Reply:
that looks great! i've just updated it and moved it to an inspector param for easy tweaking going forward 👍